src/components/dialogs/inputs/input-hooks.js

```diff
@@ -864,3 +736,81 @@ export const useCSVReader = ({ label, header }) => {
     }, [selectedFile, intl, header]);
     return [selectedFile, setSelectedFile, field, fileError];
 };
+
+export const useRadioValue = ({
+    label,
+    possibleValues = [],
+    defaultValue,
+    id,
+    validation = {},
+    inputForm,
+    doTranslation = true,
+}) => {
+    const [value, setValue] = useState(defaultValue);
+    const intl = useIntl();
+
+    useEffect(() => {
+        // Updates the component when the correct default value is given by the parent component.
+        if (defaultValue !== undefined && defaultValue.length > 0) {
+            setValue(defaultValue);
+        }
+    }, [defaultValue]);
+
+    useEffect(() => {
+        function validate() {
+            return true;
+        }
+
+        inputForm.addValidation(id ? id : label, validate);
+    }, [label, validation, inputForm, value, id]);
+
+    const handleChangeValue = useCallback((event) => {
```

Comment:
Why to use a useCallback here ?

Reply:
Good point, it is unecessary. Removed it for a lambda inline.